news.js

```diff
@@ -0,0 +1,47 @@
+const readline = require('readline');
+
+const rl = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout
+});
+
+
+var Request = require("request");
+var answer = 0;
+rl.question('Select your preferred news source by inserting the corresponding number:\n1. BBC NEWS \n2. FOX NEWS \n3. AL JAZEERA \n4. CNN ', (answer) => {
+
+    if (answer == 1) {
+
+        Request.get("https://newsapi.org/v2/top-headlines?sources=bbc-news&apiKey=8c61d66d36a04d9ca70c78d63f7a1aad", (error, response, body) => {

```

Comment:
This code appears multiple times, It is better we extract it into a function i.e
Create a function that takes a news source and returns the top 10 headlines for that source
```js
function getTopHeadlines(newsSource){
function getTopHeadlines(newsSource) {
  // how you can combine strings using template literals
  const url = `https://nap.org?sources=${newsSource}`;
  Request.get(url, () => {
    // do your stuff here
  });
}

// You can later on go ahead and call this function for every news source like
getTopHeadlines("abc-news");
```